### ISSUE TITLE
`rmlint -o summary` returns ` "size": 0` for files that are not empty

### ISSUE THREAD

**Issue by erikschul** [sahib/rmlint#680]
I'm using rmlint to find and remove duplicates between backups. They are 90% identical, but rmlint found no duplicates, but lots of "suspicious files".

It seems that if a filename has a space in it, the filesize is reported as ` "size": 0`.
I'm wondering if the problem is that the filename contains a unicode space or something like that.
If I copy the file into a new filename `test1` and `test2`, the size is correct.

Any thoughts on what's going on?

**Comment by erikschul** [comment]
It seems to be related to `badids`.
The files are owned by a uid that doesn't exist, and are therefore not compared.

## Example

Files owned by non-existing uid are "size 0":

```
cat rmlint.json | grep size
  "size": 0,
  "size": 0,
  "size": 4,
  "size": 4,
  "size": 33672,
  "size": 33672,
  "total_lint_size": 33676
```

Changing owner:
```
chown root:root -R .
```

```
rmlint a b
cat rmlint.json | grep size
  "size": 4,
  "size": 4,
  "size": 33672,
  "size": 33672,
  "size": 33672,
  "size": 33672,
  "total_lint_size": 101020
```

```
chown 1000:1000 -R .
```
```
rmlint -T "all -badids" a b
cat rmlint.json | grep size
  "size": 67348,
  "size": 67348,
  "size": 33672,
  "size": 33672,
  "total_lint_size": 101020
```

The solution is therefore to either chown everything, or to disable the `badids` check.
